test/units/modules/storage/netapp/test_na_ontap_ipspace.py

```diff
@@ -0,0 +1,138 @@
+# (c) 2018, NTT Europe Ltd.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+""" unit test for Ansible module: na_ontap_ipspace """
+
+from __future__ import print_function
+import json
+import pytest
+
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+import ansible.module_utils.netapp as netapp_utils
+from ansible.modules.storage.netapp.na_ontap_ipspace \
+    import NetAppOntapIpspace as my_module
+from units.compat import unittest
+from units.compat.mock import patch
+
+if not netapp_utils.has_netapp_lib():
+    pytestmark = pytest.skip('skipping as missing required netapp_lib')
```

Comment:
Can you change pytest.skip to pytest.mark.skip  ?

(We'll need to change it in all modules).

Reply:
updated